migrations/001_create_tables.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE User (
+    id int NOT NULL AUTO_INCREMENT,
+    login varchar(255) NOT NULL,
```

Comment:
It is not a base rule that has to be done.
But it's better when you have one style for all types etc.

So if you write NOT NULL, please use upper case for all types and keywords. Like INT and VARCHAR